### DISCUSSION TITLE
给断供烂尾楼业主再支一歪招

### DISCUSSION THREAD

**Discussion by duhanmin** [WeNeedHome/SummaryOfLoanSuspension#279] [Ideas]
![image](https://user-images.githubusercontent.com/28647031/178912273-ff0fafe6-91b5-4ced-90a5-fad4334aea7d.png)

**Comment by Leon0312** [comment]
放款有风控机制的。不可行

**Comment by bobo418** [comment]
这应该很快会被关小黑屋了吧

**Reply by 306732407** [reply]
这里很稳了，只要不被抓住项目所有人，这个所有人貌似昨天才注册的账号，希望别是国内邮箱，哈哈哈

**Comment by cccp057** [comment]
你这玩意是诈骗啊，除非润出去。但是你润出去的话，这点钱也不够生活的

**Reply by xiejun946** [reply]
哈哈哈

**Comment by startong0321** [comment]
很明显行不通的，假设你办了10张信用卡，只要你办理了一次贷款，第二次申请贷款的时候，银行就会根据你的征信，消费水平进行评估，如果银行认为你有还款能力才会放款给你，如果银行觉得你不具备还款能力，肯定是不会放款的，也就是说你前面几次可能可以申请成功，再后面的贷款申请，会基于你现在的贷款额度以及还款能力评估，很难放款的，，，

**Comment by rootkiter** [comment]
肯定行不通的。。信用卡申请机制是联网的。。。频繁申请信用卡，会降低信用卡额度

**Comment by JACWATA** [comment]
你这是馊主意啊

**Comment by edisom16** [comment]
最彻底的办法就是必须坚持交房验房合格之后再还贷，其他的没大用。只要坚持住就能胜利
